src/components/EditHabit/index.tsx

```diff
@@ -51,14 +53,18 @@ const EditHabit = ({ habit, refetch }) => {
         name: habit.name,
         type: habit.type,
         recurrence: habit.recurrence,
+        trainedFor: habit.trainedFor,
+        links: habit.links,
       }}
       onSubmit={submitUpdatedHabit}
+      render={props => <EditHabitForm {...props} />}
```

Comment:
Just be aware I moved the `EditHabitForm` in between the Formik component as I was having problems putting it in the render prop like you are doing here. But if it works now thats fine I didn't quite narrow down exactly why it was a problem.